R/plot.R

```diff
@@ -479,16 +484,29 @@ plot_predictions <- function(data,
     plot <- plot +
       geom_point(
         data = truth_data,
-        aes(y = true_value, colour = "actual"),
+        show.legend = FALSE,
+        inherit.aes = FALSE,
+        aes(y = true_value, x = .data[[x]]),
```

Comment:
```suggestion
        aes(x = .data[[x]], y = true_value),
```

so it is in the same order as L497, thus making it simpler to read & understand